GVFS/GVFS.Common/InstallerPreRunChecker.cs

```diff
@@ -45,15 +45,6 @@ public bool TryRunPreUpgradeChecks(GitVersion gitVersion, out string error)
             }
 
             this.tracer.RelatedInfo("Okay to run GVFS upgrade.");
-
-            this.tracer.RelatedInfo("Checking if Git upgrade can be run on this machine.");
-            if (gitVersion != null && !this.IsGitUpgradeAllowed(gitVersion, out error))
-            {
-                this.tracer.RelatedError($"{nameof(TryRunPreUpgradeChecks)}: {error}");
-                return false;
-            }
-
-            this.tracer.RelatedInfo("Okay to run Git upgrade.");
             this.tracer.RelatedInfo("Successfully finished pre upgrade checks.");

```

Comment:
Should we keep both of these trace statements?

```
this.tracer.RelatedInfo("Okay to run GVFS upgrade.");
this.tracer.RelatedInfo("Successfully finished pre upgrade checks.");
```

Or can they be collapsed them into one?